# This is a a cheap way to avoid errors regarding camera_enabled()
# which is apparently unimplemented anyway
diff -Naur1 libchromeos/chromeos/policy/device_policy_impl.cc libchromeos-camera-proto/chromeos/policy/device_policy_impl.cc
--- libchromeos/chromeos/policy/device_policy_impl.cc	2013-10-17 05:50:44.505004668 +0000
+++ libchromeos-camera-proto/chromeos/policy/device_policy_impl.cc	2013-10-17 06:05:44.795005098 +0000
@@ -164,5 +164,2 @@
 bool DevicePolicyImpl::GetCameraEnabled(bool* camera_enabled) const {
-  if (!device_policy_.has_camera_enabled())
-    return false;
-  *camera_enabled = device_policy_.camera_enabled().camera_enabled();
   return true;
